sklearn/cluster/mean_shift_.py

```diff
@@ -158,8 +175,9 @@ def mean_shift(X, bandwidth=None, seeds=None, bin_seeding=False,
             if len(points_within) == 0:
                 break  # Depending on seeding strategy this condition may occur
             my_old_mean = my_mean  # save the old mean
-            my_mean = np.mean(points_within, axis=0)
-            # If converged or at max_iter, adds the cluster
+            my_mean = _kernel_update(my_old_mean, points_within, bandwidth,
```

Comment:
I thought if kernel="rbf" we wanted to make the default bandwidth be dependent on ``gamma``, right?